src/dispatch/static/dispatch/src/app/store.js

```diff
@@ -65,7 +65,7 @@ const mutations = {
     state.loading = value
   },
   RESET_SNACKBAR(state) {
-    state.snackbar = Object.assign(state.snackbar, getDefaultSnackbarState())
+    state.snackbar = Object.assign({}, getDefaultSnackbarState())
```

Comment:
@kevgliss once I fixed the close button (see change in line 39), an uninitialized snackbar would appear after closing the snackbar with the error message. This change fixes the issue, but I'm not sure it's the best way.

Reply:
I don't think we need `Object.assign` here. It could just be `state.snackbar = getDefaultSnackbarState()`